src/main/scala/org/broadinstitute/dsde/workbench/sam/dataAccess/PostgresAccessPolicyDAO.scala

```diff
@@ -1350,7 +1350,59 @@ class PostgresAccessPolicyDAO(protected val writeDbRef: DbReference, protected v
     })
   }
 
-  private def setPolicyIsPublicInternal(policyPK: PolicyPK, isPublic: Boolean)(implicit session: DBSession): Int = {
+  override def recreateEffectivePolicyRolesTableEntry(resourceTypeName: ResourceTypeName, samRequestContext: SamRequestContext): IO[Unit] = {
+    val resource = ResourceTable.syntax("resource")
```

Comment:
I don't actually think that's more helpful. I don't think it is any less helpful, but verbosity has its own cost. My inclination is to leave it. Unfortunately I don't think there's a more descriptive name, because it kind of does just describe the resource.